src/app/clusters/software-diagnostics-server/software-fault-listener.cpp

```diff
@@ -15,35 +14,27 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-
-#pragma once
-
-#include <app-common/zap-generated/cluster-objects.h>
-#include <app/CommandResponseHelper.h>
-#include <platform/GeneralFaults.h>
+#include <app/clusters/software-diagnostics-server/software-fault-listener.h>
 
 namespace chip {
 namespace app {
 namespace Clusters {
+namespace SoftwareDiagnostics {
+namespace {
+SoftwareFaultListener * gListener = nullptr;
```

Comment:
@bzbarsky-apple @ksperling-apple similar question here: this is a global thing ... but given it is 0-initialized we probably do not need `Global` (as it has no constructor/destructor call). Is that true?